www/_posts/2021-08-25-translations.md

```diff
@@ -0,0 +1,26 @@
+---
+layout: post
+author:
+    name: Jesse MacFadyen
+    url: https://twitter.com/purplecabbage
+title:  "Declaring Translation Bankruptcy"
+categories: announcements
+tags: news
+---
+
+
+It has been talked about for years, and even though we [agreed to do it](https://lists.apache.org/thread.html/4768b193b66d75d1c0c7c08303c54eb0fa15894c60e99b607f7e3e5d%40%3Cdev.cordova.apache.org%3E), we never really moved on it.
+Taking things away is always harder than adding things.
+
+Back in the day, we had active contributions from the community to translate our documentation, and we had a pretty robust system to pull them all together and publish the html site.  This system has run flawlessly for years, however, the community translations slowed, then stopped, while the system still carefully churned through it all. We have been providing translated docs since Cordova CLI 3.1.0 while the maintenance of these translated docs started to diverge around Cordova CLI 6.x
+
+Over the next few weeks we will be making aggessive changes to the documentation system.
```

Comment:
Maybe put this line up front to make it clear from the start what this is about.